server/controllers/user.js

```diff
@@ -21,3 +22,38 @@ exports.searchUsers = asyncHandler(async (req, res, next) => {
 
   res.status(200).json({ users: users });
 });
+
+exports.createBoard = asyncHandler(async (req, res) => {
+  const user = await User.findById(req.user.id);
+
+  const newBoard = await Board.create({
+    name: "My board",
+    columns: [
+      {
+        name: "In progress",
+        cards: [],
+        createdAt: Date.now(),
+      },
+      {
+        name: "Completed",
+        cards: [],
+        createdAt: Date.now(),
+      },
+    ],
+    user: user,
+  });
+
+  const boardWithoutPassword = newBoard.removePassword();
+  return res.status(200).json({ boards: boardWithoutPassword });
+});
+
+exports.getUserBoards = asyncHandler(async (req, res, next) => {
+  const user = await User.findById(req.user.id);
+  const boards = await Board.find({ "user._id": user._id });
+
+  const boardsWithoutPassword = boards.map((board) => {
```

Comment:
just for shorthand

```suggestion
  const boardsWithoutPassword = boards.map(board => board.removePassword())
```

I think for a straight forward method like this wouldn't be worth taking 3 lines of code :) just my opinion though

Reply:
I agree with the implicit return shorthand!

Reply:
Thank you